### PR TITLE
[WIP] Make more emojis work for PY2 🐯

### PR DESCRIPTION
**Pull request by akelad**
**Proposed changes**:
- More emoji support including but not limited to: 🧟‍♂️ 🧞‍♂️ 🧖‍♀️

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog

### CODE REVIEW

File: rasa_core/utils.py [hunk 1]
```diff
@@ -293,7 +293,8 @@ def construct_yaml_str(self, node):
     if sys.maxunicode == 0xffff:
         yaml.reader.Reader.NON_PRINTABLE = re.compile(
                 '[^\x09\x0A\x0D\x20-\x7E\x85\xA0-\uD7FF\ud83d\uE000-\uFFFD'
-                '\ude00-\ude50]')
+                '\ude00-\ude50-\udc4d\ud83c\udf89\ude80\udc4c\ud83e\uddde'
```
Comment (tmbo) [inline rasa_core/utils.py]: this is a bit of an odd syntax - usually the pattern is: `[A-Za-z]` but now there are two dashes: `\ude00-\ude50-\udc4d` does that work?

Comment (akelad) [reply rasa_core/utils.py]: haha that was unintentional, idk how that worked tbh. i'll get rid of the second dash

**Comment by tmbo** [comment]
@akelad can you do the fix & add to the readme - then we can include this in the minor release

**Comment by akelad** [comment]
yes sorry will do in a sec, what do you mean add to the Readme though? I assume changelog?

**Comment by tmbo** [comment]
Yes thats what I meant :D

**Pull request merged**